.github/workflows/codeql-push.yml

```diff
@@ -73,10 +73,12 @@ jobs:
         trap-caching: false
 
     - name: Install deps
-      if: ${{ env.BUILD_TARGETS != '' }}
+      # if: ${{ env.BUILD_TARGETS != '' }}
```

Comment:
need to remove this before we can land

relatedly - im starting to think that just downgrading the host is simpler